src/components/Primer/Primer.tsx

```diff
@@ -55,7 +47,20 @@ const Primer = ({ selectPaymentMethod, onSubmit }: PrimerProps) => {
 
   if (isLoading) return <Loader />;
 
-  return <div id={CONTAINER} />;
+  return (
+    <>
+      <PrimerContainer id={CONTAINER} />
+      {isMyAccount && primerCheckout?.submit && (
+        <UpdateButtonStyled
+          disabled={isButtonDisabled}
+          onClick={() => primerCheckout?.submit()}
```

Comment:
```suggestion
          onClick={primerCheckout?.submit}
```
this way you don't recreate the function on each render

Reply:
also, you can get rid of the optional chaining, as you check for the function's exitence earlier